packages/tupaia-web/src/features/Map/MapLegend/MobileMapLegend.tsx

```diff
@@ -60,20 +62,22 @@ export const MobileMapLegend = ({ children }: { children: ReactNode }) => {
     setExpanded(!expanded);
   };
   return (
-    <Wrapper>
-      {expanded ? (
-        <ExpandedLegend>
-          <CloseButton onClick={toggleExpanded} aria-label="Close legend">
-            <Close />
-          </CloseButton>
-          {children}
-        </ExpandedLegend>
-      ) : (
-        <MapLegendExpandButton onClick={toggleExpanded}>
-          <ExpandIcon />
-          Map Legend
-        </MapLegendExpandButton>
-      )}
-    </Wrapper>
+    <ErrorBoundary>
```

Comment:
If we have an error boundary in MapLegend.tsx then we shouldn't need this one as well